src/utils/signaling.js

```diff
@@ -849,12 +846,37 @@ Signaling.Standalone.prototype.helloResponseReceived = function(data) {
 			return
 		}
 
+		this.helloResponseErrorCount++
+
+		if (this.signalingConnectionError === null && this.helloResponseErrorCount < 5) {
```

Comment:
so we show this error 5 times?

Reply:
It will be shown only if `this.signalingConnectionError === null`. So either the same message was already being shown (because first [there was an error in the websocket](https://github.com/nextcloud/spreed/blob/24c9e2b158ff90c2010401542557ffb4793c7aa8/src/utils/signaling.js#L599)) or it will be shown once here.

The message is only shown again at the fifth error in order to change its text (I did not find how to modify the text of a toast already being shown).

In any case this is part of the second commit that could be a good idea to drop, so keep that in mind :-)